ruby/vampires.rb

```diff
@@ -0,0 +1,91 @@
+puts "How many employees will be processed"
+employees = gets.chomp
+employees = employees.to_i
+
+until employees == 0
+
+	puts "What is your name?"
+	name = gets.chomp
+	name = name.to_s
+
+	puts "How old are you?"
+	age = gets.chomp
+	age = age.to_i
+
+	puts "What year were you born?"
+	year_of_birth = gets.chomp
+	year_of_birth = year_of_birth.to_i
+
+	if 2017 - year_of_birth == age

```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`